holonix/legacy/default.nix

```
@@ -0,0 +1,15 @@
+let
+  holonixPath = ../.; # points to the current state of the Holochain repository
+  holonix = import holonixPath {
+    holochainVersionId = "v0_1_0"; # specifies the Holochain version
```

in case this is used?
```suggestion
    holochainVersionId = "v0_1_3"; # specifies the Holochain version
```

---

ironically this argument will be without any effect once we merge the PR